setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="OpenFisca-France-Fiscalite-Miniere",
-    version="1.1.2",
+    version="1.2.0",
```

Comment:
Comme la modification concerne l'exécution du module et que le modèle de la loi n'a pas été modifié, modifier le bump intermédiaire vers un bump de patch après rebase ?

Reply:
Après rebase sur `master` :
```suggestion
    version="1.1.4",
```